DigitalLearningSolutions.Web/Views/CookieConsent/CookieConfirmation.cshtml

```diff
@@ -0,0 +1,22 @@
+﻿@{
+    ViewData["Title"] = "Digital Learning Solutions - Cookie confirmation";
+    ViewData["DoNotDisplayNavBar"] = true;
+}
+
+<div class="nhsuk-width-container app-width-container">
+    <div class="nhsuk-grid-row">
+        <div class="nhsuk-grid-column-full">
+            <div class="nhsuk-u-reading-width">
+                <h1>Your cookie settings have been saved</h1>
+                <p>We'll save your settings for a year.</p>
+                <p>We'll ask you if you're still OK with us using cookies when either:</p>
+                <ul>
+                    <li>it's been a year since you last saved your settings</li>
+                    <li>we add any new cookies or change the cookies we use</li>
+                </ul>
+                <p>You can also <a href="@Url.Action("CookiePolicy","CookieConsent")">choose which cookies we use</a> at any time.</p>
+            </div>
+            <p class="nhsuk-body-s nhsuk-u-secondary-text-color nhsuk-u-margin-top-7 nhsuk-u-margin-bottom-0">Updated: tbc ???????</p>
+        </div>
+    </div>

```

Comment:
![image](https://user-images.githubusercontent.com/18263516/225888883-8f7be7d5-8d2c-4792-af68-09ef025416e3.png)

This will redirect to cookie policy page

Reply:
Implemented breadcrumb navigation in confirmation page